pkg/R/RDD.R

```diff
@@ -258,16 +258,16 @@ setMethod("persist",
 #' cache(rdd) # rdd@@env$isCached == TRUE
 #' unpersist(rdd) # rdd@@env$isCached == FALSE
 #'}
-setGeneric("unpersist", function(rdd) { standardGeneric("unpersist") })
+setGeneric("unpersist", function(x) { standardGeneric("unpersist") })
 
 #' @rdname unpersist-methods
 #' @aliases unpersist,RDD-method
 setMethod("unpersist",
-          signature(rdd = "RDD"),
-          function(rdd) {
-            callJMethod(getJRDD(rdd), "unpersist")
-            rdd@env$isCached <- FALSE
-            rdd
+          signature(x = "RDD"),
+          function(x) {
+            callJMethod(getJRDD(x, "unpersist"))
```

Comment:
This should be `callJMethod(getJRDD(x), "unpersist")` -- The bracket needs to move in